### ISSUE TITLE
[Bug|Feature Request] Short Description of Issue

### ISSUE THREAD

**Issue by ChuanNegro** [cobbr/Covenant#299]
BUG

In Dev branch the upload does not work properly.
When I uploading the file, the upload task finished but the file size is 0.

On any active Grunt start upload task of any file.

**Screenshots**
![image](https://user-images.githubusercontent.com/80527818/111026265-ad1e6100-83e9-11eb-9fb6-556aceeef1b0.png)
![image](https://user-images.githubusercontent.com/80527818/111026304-e0f98680-83e9-11eb-8a8d-ad1ff8843d75.png)

If you need more info, let me know please

**Comment by cobbr** [comment]
Hey @ChuanNegro I'm not able to reproduce this issue. Have you tested with a file that is not likely to get picked up by AV?

Can you share the output of the Upload task?

**Comment by ChuanNegro** [comment]
AV is turned off :
![image](https://user-images.githubusercontent.com/80527818/111136301-dcfe6d80-857d-11eb-8a79-78bf9574f598.png)
I am trying to upload the wmiexec.exe which has size around 6MB
![image](https://user-images.githubusercontent.com/80527818/111136363-f4d5f180-857d-11eb-84cd-17cc9a1888c9.png)
after I submit the upload task it finished with no errors
![image](https://user-images.githubusercontent.com/80527818/111136550-28188080-857e-11eb-81b1-0bc44af49504.png)
but the file does not have a proper size:
![image](https://user-images.githubusercontent.com/80527818/111136602-35ce0600-857e-11eb-868e-9894e59669a4.png)

now it behaves that every file has same size in this case 226KB before it was 0.
If you need more details let me know what and how I can get it.
Thanks

**Comment by ChuanNegro** [comment]
I tried to get Slack client install file ~87MB
![image](https://user-images.githubusercontent.com/80527818/111138723-9100f800-8580-11eb-89d8-55e446f7e9ab.png)
I added it into the upload task
![image](https://user-images.githubusercontent.com/80527818/111138806-aa09a900-8580-11eb-9bbf-bb3ba6fe7ec5.png)
and push
result is the same size is 226KB

![image](https://user-images.githubusercontent.com/80527818/111138901-c6a5e100-8580-11eb-884e-ec0fbf6b42fd.png)
![image](https://user-images.githubusercontent.com/80527818/111138959-d6bdc080-8580-11eb-8d3a-18e1ff06e2fd.png)

![image](https://user-images.githubusercontent.com/80527818/111138889-c279c380-8580-11eb-94b2-b95c47ed10f9.png)

**Comment by cobbr** [comment]
Strange, I wonder if the problem is with large file uploads?

**Comment by ChuanNegro** [comment]
Hi,
It seems like caching the file.
I tried to upload the small PNG image. It was done correctly and I can open it on the "Victim's" workstation
after this upload I took the exe file let it upload and it has same size as the previously uploaded EXE

![image](https://user-images.githubusercontent.com/80527818/111203763-ad735380-85c5-11eb-8959-2bd5556c310b.png)

is there any, like debug which I can turn on, to give you more info ?
thanks

**Comment by cobbr** [comment]
So you are saying that it is uploading the same file each time, even after selecting different files?


Have you tried the dev branch to see if you have a different result?

**Comment by ChuanNegro** [comment]
yes, it seems that it behaves like this. This is actually happening in DEV branch .

**Comment by cobbr** [comment]
I'm wondering if this is related to #306, which has been fixed. Can you pull and re-test @ChuanNegro ? The current largest supported file size is 50MB.
